.travis.yml

```diff
@@ -7,6 +7,15 @@ cache: pip
 
 install: 'pip install pytest-cov -r requirements.txt'
 
-script: pytest --cov=.
+script:
+  # Run the tests
+  - pytest --cov=. || true  # TODO. Remove the "|| true" when there is tests
```

Comment:
We should just let the travis fail, as without any tests the CI result doesn't really give us any confirmation on quality

Reply:
Actually, here the knowledge whether requirements are valid or not, is already useful information. We had a real in-production failure because of this.

Reply:
Having just a single test doesn't confirm the quality much better. Confirmation of quality should come from the test coverage. I would suggest that you integrate to codecov.io or similar for that. Then you would get the info about (a) how well are the parts changed in the PR covered by the tests and (b) how well is the project tested over all.